docs/ref/contrib/admin/index.txt

```diff
@@ -1939,6 +1988,30 @@ return the uncompressed versions of the various JavaScript files, including
 
 .. _jQuery: https://jquery.com
 
+.. _contrib-admin-select2:
+
+jQuery Select2
```

Comment:
I'm somewhat hesitant to document this. I'd rather have some flexibility in changing the implementation in the future rather than make a long-term commitment to Select2... any thoughts about that?